README.md

```diff
@@ -1,5 +1,16 @@
-# docs-community
+# Python Docs WG
 
 [![Documentation Status](https://readthedocs.org/projects/docs-community/badge/?version=latest)](https://docs-community.readthedocs.io/en/latest/?badge=latest)
- 
-Community management for documentation contributors and the Docs Workgroup
+
+The PSF Documentation Working Group (Docs WG) was chartered by the PSF to help bootstrap a larger Python documentation team and community.
+This repo serves as documentation for the Docs WG, which is funny, in a recursive kind of way.
+
+For example, to find out more about us and what we do, [read the docs](https://docs-community.readthedocs.io/en/latest/).
+
+## Developer Info
+
+1. Clone the repo.
+2. Make a virtualenv.
+3. `pip install -r requirements.txt`
+4. `sphinx-autobuild docs docs/_build`
```

Comment:
~~~suggestion
## Build docs and view changes

1. Clone this repo

   ```console
   git clone https://github.com/python/docs-community.git
   cd docs-community
   ```

2. Create a new Python virtual environment

   ```console
   python -m venv docs-wg-env
   ```

3. Activate the environment ([platform/shell-specific](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments))

4. Install dependencies

   ```console
   python -m pip install -r requirements.txt
   ```
5. Build the docs, open them in your browser and update whenever changes are made

   ```console
   sphinx-autobuild --open-browser docs docs/_build
   ```
~~~

Make this section much clearer, more descriptive and more useful

Reply:
I've always struggled with this with venv (since like you, most of my work uses conda envs which by design use descriptive names, whereas venv names are all over the place—some use descriptive names, while some use one of perhaps a half dozen different "standard names", e.g. `venv`, `env`, `.venv`, `.env`, etc). The docs don't formally recommend anything, some venv/virtualenv-based systems (`virtualenvwrapper`, `virtualenvhelper`, `mkvirtualenv`, etc, etc.) do use centralized names, and the name gets injected into the prompt, where it is helpful to be descriptive to confirm the user is in the venv they expect (as I've made that mistake before).

As such, I went with a descriptive name here, but I'm happy to defer to something else if someone who knows better than me (which likely includes most if not all of you) thinks so.